controller/scripts/lint.ini

```diff
@@ -23,6 +23,7 @@ ignore-patterns=csi_pb2.*[.]py
 # relevant categories: E - Error, W - Warning, R - Refactor, C - Convention
 disable=
  # easy:
+ raise-missing-from, # W - add "from" after some of the "raise"
```

Comment:
forgot to mention that IMO it's worth adding **`happened only on Jenkins,`**`add "from"...`

Reply:
as far as we know. I dont think its a fact that it cant happen in other places

Reply:
that's why it's in past tense.
should be useful for reproduction purposes

Reply:
done